shelldio.sh

```diff
@@ -127,6 +150,11 @@ remove_station() {
 	fi
 }
 
+reset_favorites() {
+	echo "Πραγματοποιείται διαγραφή του αρχείου αγαπημένων.."
```

Comment:
Ισως θα μπορούσε να βελτιωθεί λίγο αυτό ως προς το user-friendliness. Μία πρόταση:

Επειδή πρόκειται για διαγραφή ενός μεγάλου έρωτα ραδιοφωνικών σταθμών (lol), ίσως να ρωτήσουμε τον/ην _user_ να το ξανασκεφτεί λιγάκι:

```bash
while true; do
    read -p "Θες σίγουρα να διαγράψεις το αρχείο αγαπημένων;" yn
    case $yn in
        [Yy]* ) rm -f "$my_stations"; break;;
        [Nn]* ) exit;;
        * ) echo "Παρακαλώ απαντήστε με ναί ή όχι (yes/no)";;
    esac
done
````

Αν επιλέξει `yes` τότε ελέγχουμε αν πήγαν τα πράγματα όπως τα περίμενε και ενημερώνουμε κατάλληλα:

```bash
reset_favorites() {
         rm -f "$my_stations"
         if [ -f "$my_stations" ]; then
	     echo "Απέτυχε η διαγραφή του αρχείου αγαπημένων"; exit 1
         done
         echo "Το αρχείου αγαπημένων διαγράφτηκε επιτυχώς"; exit 0
```

WDYT ?

Reply:
For sure! Ωραία ιδέα!